core/src/test/java/org/elasticsearch/update/UpdateIT.java

```diff
@@ -968,7 +977,9 @@ public void run() {
                 }
 
             };
-            new Thread(r).start();
+            Thread client = new Thread(r);
+            client.setName("UpdateIT-Client-" + i);
```

Comment:
Naming a thread client is a bit confusing because `client()` is a static method commonly used in tests to fetch the `Client`. So maybe `updater`?